msticpy/context/azure/sentinel_core.py

```diff
@@ -100,15 +111,17 @@ def __init__(
         self._token: Optional[str] = None
 
         workspace = kwargs.get("workspace", ws_name)
-        self._default_workspace: Optional[str] = workspace
+        self._default_workspace_name: Optional[str] = workspace
         self.workspace_config = WorkspaceConfig(workspace)
 
         logger.info("Initializing Microsoft Sentinel connector")
         logger.info(
-            "Params: Cloud=%s; ResourceId=%s; Workspace=%s",
+            "Params: Cloud=%s; ResourceId=%s; Workspace=%s, Subscription=%s, ResourceGroup=%s",
```

Comment:
Actually, logger docs say *not* to use f-strings. The reason being (I think) is because f-strings are always evaluated. The logger substitutions are only evaluated if the logger level is at the level to trigger the call. It essentially makes uncalled logging calls cost-free.